educator/src/Dscp/Educator/Config.hs

```diff
@@ -39,6 +40,10 @@ type EducatorConfig = WitnessConfig ++
         , "certificates" ::<
            '[ "latex" ::: FilePath
             , "resources" ::: FilePath
+            , "issuer" ::<
+               '[ "name" ::: ItemDesc
+                , "url" ::: ItemDesc
```

Comment:
@flyingleafe this was called "url" to match `CertificateIssuerInfo`, would you prefer this to be "website" too?

Reply:
@pasqu4le yes, let it be consistent.